### ISSUE TITLE
Tab bar should be hidden when there is only one tab

### ISSUE THREAD

**Issue by atirut-w** [rafaelmardojai/firefox-gnome-theme#315]

**Comment by CleoMenezes** [comment]
Please, [check this out](https://github.com/rafaelmardojai/firefox-gnome-theme#features).

**Comment by rafaelmardojai** [comment]
> Now I feel pretty dumb for not reading all of the readme but can this be enabled by default pls?

Not really, it also requires you to manually move buttons out of the tabs bar. Also you will lost Firefox functionality like dragging tabs between windows when there's only one. That's why it is behind an optional feature.
